sdk/src/main/scala/com/horizen/account/mempool/MempoolMap.scala

```diff
@@ -0,0 +1,149 @@
+package com.horizen.account.mempool
+
+import com.horizen.SidechainTypes
+import com.horizen.account.transaction.EthereumTransaction
+import scorex.util.{ModifierId, ScorexLogging}
+
+import java.math.BigInteger
+import scala.collection.concurrent.TrieMap
+import scala.collection.mutable
+import scala.util.Try
+
+case class MempoolMap() extends ScorexLogging {
+
+  val all = TrieMap[ModifierId, SidechainTypes#SCAT]()
+  val executableTxs = TrieMap[SidechainTypes#SCP, scala.collection.mutable.SortedMap[BigInteger, ModifierId]]()
+  val nonExecutableTxs = TrieMap[SidechainTypes#SCP, scala.collection.mutable.SortedMap[BigInteger, ModifierId]]()
+  val nonces = TrieMap[SidechainTypes#SCP, BigInteger]()
```

Comment:
let's add some description to explain the context of these fields.
It is not clear the meaning of `scala.collection.mutable.SortedMap[BigInteger, ModifierId]` for example.

Reply:
Done